pkg/apis/query/v1alpha1/cache_types.go

```diff
@@ -0,0 +1,151 @@
+package v1alpha1
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	// ResourceKindClusterCache is kind name of ClusterCache.
+	ResourceKindClusterCache = "ClusterCache"
+	// ResourceSingularClusterCache is singular name of ClusterCache.
+	ResourceSingularClusterCache = "clusterCache"
+	// ResourcePluralClusterCache is plural name of ClusterCache.
+	ResourcePluralClusterCache = "clusterCaches"
+	// ResourceNamespaceScopedClusterCache indicates if ClusterCache is not NamespaceScoped.
+	ResourceNamespaceScopedClusterCache = false
+)
+
+// +genclient
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +kubebuilder:subresource:status
+// +kubebuilder:resource:shortName=cc
```

Comment:
Is it a `namespace-scoped` resource or `cluster-scoped`? 
If no specific instructions, it takes `namespace-scoped` by default.

I think it should be a `cluster-scoped` resource.
```suggestion
// +kubebuilder:resource:scope="Cluster"
```
Suggest add the `scope` and removes shortName (We can add it back later. )

Reply:
ok, add the scope and remove shortName.